### PR TITLE
style: Move font-feature-settings outside of mako

### LINKED ISSUE #19290: Move font-feature-settings outside of mako.
I'll take this one as a sub-issue of #19015

### PR DESCRIPTION
**Pull request by CYBAI**
This is a sub-PR of #19015 
r? emilio 

---
- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #19290 
- [x] These changes do not require tests

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19293)

### CODE REVIEW

File: components/style/values/computed/font.rs [hunk 1]
```diff
@@ -286,6 +287,17 @@ pub type FontVariantLigatures = specified::VariantLigatures;
 /// Use VariantNumeric as computed type of FontVariantNumeric
 pub type FontVariantNumeric = specified::VariantNumeric;
 
+/// Use FontSettings as computed type of FontFeatureSettings
+pub type FontFeatureSettings = FontSettings<FontSettingTagInt>;
+
+impl FontFeatureSettings {
+    #[inline]
+    /// Default value of `font-feature-settings` as `normal`
+    pub fn normal() -> FontFeatureSettings {
```
Comment (emilio) [inline components/style/values/computed/font.rs]: Given this is a `FontSettings` variant, we could try to move this to `FontSettings`, but that can be a followup PR.

**Review by emilio — APPROVED** [review]
Nice!

File: components/style/values/computed/font.rs (hunk 1 above)
Comment (CYBAI) [reply components/style/values/computed/font.rs]: Great! It should be moved under `FontSettings`; after this PR merged, I'll send one to fix it!

**Pull request merged**